enatega-multivendor-restaurant/src/screens/Setting/SelectLanguage.js

```diff
@@ -46,14 +47,18 @@ export default function SelectLanguage() {
   ]
 
   const changeLanguage = async(language) => {
+    setLoader(true)
     console.log(language)
     setSelectedLanguage(language);
+    await i18next.reloadResources(language, null);
     i18next.changeLanguage(language)
     if (Platform.OS === 'android') {
       await AsyncStorage.setItem('enatega-language', language);
     }
+
```

Comment:
@Romana-Aijaz please check for ios. change language -> close app, then reload app.